google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2574,9 +2743,16 @@ enum Position {
   @GuardedBy("lock")
   private final Queue<WaiterFuture> waiters = new LinkedList<>();
 
+  @GuardedBy("lock")
+  private final Queue<MultiplexedSessionWaiterFuture> multiplexedSessionWaiters =
+      new LinkedList<>();
+
   @GuardedBy("lock")
   private int numSessionsBeingCreated = 0;
 
+  @GuardedBy("lock")
+  private int numMultiplexedSessionsBeingCreated = 0;
```

Comment:
Change to a boolean to make it clear that we only have 1 multiplexed session. This value would otherwise only ever by 0 or 1.